test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -341,7 +341,9 @@ public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuild
         $returnValue = $null
         try
         {
-            $returnValue = Get-CimClassPropVal Win32_DeviceGuard $propertyName -namespace 'root\Microsoft\Windows\DeviceGuard' -ErrorAction Stop
+            #Since we want to catch any exceptions here, we won't use the above Get-Cim* helper functions
+            $inst = Get-CimInstance Win32_DeviceGuard -namespace 'root\Microsoft\Windows\DeviceGuard' -ErrorAction Stop
+            $returnValue = $inst.$propertyName
         }
         catch
```

Comment:
since the original issue was that this namespace didn't exist on a specific device/VM, should we only catch the `not found` exception rather than everything?